waku/waku_archive_legacy/driver.nim

```diff
@@ -41,6 +41,7 @@ method getMessagesV2*(
     endTime = none(Timestamp),
     maxPageSize = DefaultPageSize,
     ascendingOrder = true,
+    requestId: string,
```

Comment:
If requestId is mandatory here, would it be better at the front of the args?
Maybe it is just my old habit, I know it works in nim though...

Reply:
> If requestId is mandatory here, would it be better at the front of the args? Maybe it is just my old habit, I know it works in nim though...

Ah yes you are right! Nevertheless let me take the license to leave it as is given that we will deprecate this code soon. Thanks for the comment!